docs/modules/ROOT/pages/index.adoc

```diff
@@ -4,6 +4,14 @@ This is an operator for Kubernetes that can manage https://www.openpolicyagent.o
 
 WARNING: This operator only works with images from the https://repo.stackable.tech/#browse/browse:docker:v2%2Fstackable%2Fopa[Stackable] repository
 
+== Supported Versions
+
+The Stackable Operator for OPA currently supports the following versions of OPA:
```

Comment:
Are you submitting a PR to the docs master list to include OPA?

Reply:
https://github.com/stackabletech/documentation/pull/138 done